--- a/ar.c
+++ b/ar.c
@@ -260,5 +260,5 @@
 #endif
-    test_data = (void*) (((u32) &test_data_pad + 0x1F) & 0xFFFFFFE0);
-    dummy_data = (void*) (((u32) &dummy_data_pad + 0x1F) & 0xFFFFFFE0);
-    buffer = (void*) (((u32) &buffer_pad + 0x1F) & 0xFFFFFFE0);
+    test_data = (u32*) (uintptr_t) (((u32) &test_data_pad + 0x1F) & 0xFFFFFFE0);
+    dummy_data = (u32*) (uintptr_t) (((u32) &dummy_data_pad + 0x1F) & 0xFFFFFFE0);
+    buffer = (u32*) (uintptr_t) (((u32) &buffer_pad + 0x1F) & 0xFFFFFFE0);
     for (i = 0; i < 8; i++) {
